<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from edumip_my_robot.xacro          | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="edumip_my_robot" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <material name="red">
    <color rgba=".8 0 0 1"/>
  </material>
  <material name="blue">
    <color rgba="0 0 .8 1"/>
  </material>
  <material name="green">
    <color rgba="0 .8 0 1"/>
  </material>
  <link name="edumip_body">
    <visual>
      <geometry>
        <box size="0.035 0.05 0.1"/>
      </geometry>
      <material name="green"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
    </visual>
  </link>
  <link name="wheelL">
    <visual>
      <geometry>
        <cylinder length="0.01" radius="0.034"/>
      </geometry>
      <material name="red"/>
      <origin rpy="1.57075 0 0" xyz="0 0 0"/>
    </visual>
  </link>
  <link name="wheelR">
    <visual>
      <geometry>
        <cylinder length="0.01" radius="0.034"/>
      </geometry>
      <material name="blue"/>
      <origin rpy="1.57075 0 0" xyz="0 0 0"/>
    </visual>
  </link>
  <link name="range_finder">
    <visual>
      <geometry>
        <cylinder length="0.01" radius="0.005"/>
      </geometry>
      <material name="red"/>
      <origin rpy="0 0 0" xyz="0 0 0.05"/>
    </visual>
  </link>
  <joint name="RangeFix" type="fixed">
    <parent link="edumip_body"/>
    <child link="range_finder"/>
    <origin xyz="0.03 0 0" rpy="0 0 0"/>
  </joint>
  <joint name="jointL" type="continuous">
    <parent link="edumip_body"/>
    <child link="wheelL"/>
    <origin xyz="0 0.035 -0.05"/>
    <axis xyz="0 1 0"/>
  </joint>
  <joint name="jointR" type="continuous">
    <parent link="edumip_body"/>
    <child link="wheelR"/>
    <origin xyz="0 -0.035 -0.05"/>
    <axis xyz="0 1 0"/>
  </joint>
</robot>

